### PR TITLE
Update German stringtable.xml + BEFilter Fix for Apex +Update ApexContent/Config_Vehicles.hpp

### PR DESCRIPTION
**Pull request by WolfSkin1337**
<Please review the guidelines for contributing to this repository. The link is above.>

Resolves #185. <If applicable.>

#### Changes proposed in this pull request: 
- Update German stringtable.xml
- BEFilter Fix for Apex
- Update ApexContent/Config_Vehicles.hpp

### CODE REVIEW

**Comment by WolfSkin1337** [comment]
Can you tell me why dublicate?
And How can I change this?
Help me please to understand this.

**Comment by Jawshy** [comment]
Calm down! It's just because https://github.com/AsYetUntitled/Framework/pull/185 and this PR are the same. This is just waiting to be reviewed and then merged.

**Comment by WolfSkin1337** [comment]
I have closed the #185 and delete the String on my Fork.

ok good to hear.

**Comment by baua1310** [comment]
@WolfSkin1337 have you finished all your work? Then i gonna check and review everything after christmas.

**Comment by WolfSkin1337** [comment]
Tomorrow i do changes because at Review on my one server for German the text are too long.

**Comment by WolfSkin1337** [comment]
@baua1310 
yes its the end!

**Comment by WolfSkin1337** [comment]
+ BEFilter Fix for Apex

**Comment by WolfSkin1337** [comment]
oh man I will do nothing more

**Comment by WolfSkin1337** [comment]
so sry for that.
review it please and merged when ok, because i can clean my fork for next content.

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -4459,7 +4459,7 @@
             <Czech></Czech>
             <Spanish>Costo de Transmisión: %1&lt;br /&gt;Siguiente Transmisión Disponible en: %2</Spanish>
             <Russian></Russian>
-            <German></German>
+            <German>Broadcastkosten: %1&lt;br /&gt;Nächster Broadcast verfügbar in: %2</German>
```
Comment (baua1310) [inline Altis_Life.Altis/stringtable.xml]: **Übertragungskosten**: %1&lt;br /&gt;**Nächste Übertragung möglich** in: %2

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -4503,7 +4503,7 @@
             <Czech></Czech>
             <Spanish>Necesitas %1 para transmitir un anuncio!</Spanish>
             <Russian></Russian>
-            <German></German>
+            <German>Du benötigst $%1, um ein Broadcast zu senden!</German>
```
Comment (baua1310) [inline Altis_Life.Altis/stringtable.xml]: Du benötigst $%1, um **eine Nachricht senden zu können**!

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -4560,7 +4560,7 @@
             <Czech>Jste utekl z vězení, si stále zachovávají své předchozí zločiny a nyní mají počet uniknout vězení.</Czech>
             <Spanish>Te has escapado de la cárcel, retienes tus crimenes anteriores más un crimen de escapado de la cárcel.</Spanish>
             <Russian></Russian>
-            <German>Du bist aus dem Gefängnis ausgebrochen, die Polizei sucht bereits wegen deiner früheren Verbrechen und Gefängnisausbruch nach dir.</German>
+            <German>Du bist aus dem Gefängnis ausgebrochen! Die Polizei sucht dich jetzt wegen deiner früheren Verbrechen und dem Gefängnisausbruch nach dir.</German>
```
Comment (baua1310) [inline Altis_Life.Altis/stringtable.xml]: Du bist aus dem Gefängnis ausgebrochen! Die Polizei sucht dich jetzt wegen deiner früheren Verbrechen und dem Gefängnisausbruch.
without **nach dir** at the end

File: Altis_Life.Altis/stringtable.xml [hunk 4]
```diff
@@ -5958,7 +5958,7 @@
             <Polish>Ten dom jest już własnością, nawet jeśli nie należy widzieć tę wskazówkę ...</Polish>
             <Portuguese>Esta casa já é de propriedade mesmo que você não deveria estar vendo essa dica ...</Portuguese>
             <Russian>Этот дом уже находится в собственности, даже если вы не должны видеть эту подсказку ...</Russian>
-            <German>Dieses Haus befindet sich bereits im Besitz, auch wenn Sie nicht diesen Hinweis werden sehen ...</German>
+            <German>Dieses Haus befindet sich bereits in deinem Besitz, auch wenn du es nicht sehen solltest...</German>
```
Comment (baua1310) [inline Altis_Life.Altis/stringtable.xml]: Dieses Haus befindet sich bereits in deinem Besitz. **Diesen Hinweis solltest du eigentlich gar nicht sehen...**

File: Altis_Life.Altis/stringtable.xml [hunk 5]
```diff
@@ -7697,7 +7697,7 @@
             <Polish>Tylko część swoich materiałów może być przetwarzane ze względu na osiągnięcie maksymalnej wadze.</Polish>
             <Portuguese>Apenas uma parte de seus materiais puderam ser processados, devido o peso máximo ter sido atingido.</Portuguese>
             <Russian>Только часть ваших материалов может быть обработан из-за достижения вашего максимального веса.</Russian>
-            <German>Nur ein Teil der Materialien kann verarbeitet werden, da das maximale Gewicht sonst erreicht würde.</German>
+            <German>Nur ein Teil der Materialien konnte verarbeitet werden, sonst würde das maximale Gewicht erreicht.</German>
```
Comment (baua1310) [inline Altis_Life.Altis/stringtable.xml]: Nur ein Teil der Materialien konnte verarbeitet werden, sonst würde das maximale Gewicht erreicht **werden**.

**Comment by WolfSkin1337** [comment]
It is Update with the changes from @baua1310 .

So all fine in German Strings

**Comment by baua1310** [comment]
lgtm

**Comment by Jawshy** [comment]
Thanks @WolfSkin1337 and @baua1310. @BoGuu and I will try to have this tested and merged soon.

**Comment by WolfSkin1337** [comment]
#Fixes

File: ApexContent/Config_Vehicles.hpp [hunk 6]
```diff
@@ -63,6 +63,7 @@ class CarShops {
         side = "civ";
         conditions = "";
         vehicles[] = {
+
```
Comment (Jawshy) [inline ApexContent/Config_Vehicles.hpp]: :confused:

**Comment by WolfSkin1337** [comment]
i hope down?
1to1 with apex dlc

File: ApexContent/Config_Vehicles.hpp [hunk 7]
```diff
@@ -165,15 +164,8 @@ class LifeCfgVehicles {
...
-    *         FORMAT:
-    *        STRING (Conditions) - Must return boolean :
-    *            String can contain any amount of conditions, aslong as the entire
-    *            string returns a boolean. This allows you to check any levels, licenses etc,
-    *            in any combination. For example:
-    *                "call life_coplevel && license_civ_someLicense"
-    *            This will also let you call any other function.
-    *    Textures config follows { Texture Name, side, {texture(s)path}, Conditions}
+    *       1: STRING (Condition)
+    *    Textures config follows { Texture Name, side, {texture(s)path}, Condition}
```
Comment (BoGuu) [inline ApexContent/Config_Vehicles.hpp]: What happened to the comments!

Comment (WolfSkin1337) [inline ApexContent/Config_Vehicles.hpp]: see [Framework/Altis_Life.Altis/config/Config_Vehicles.hpp](https://github.com/AsYetUntitled/Framework/blob/master/Altis_Life.Altis/config/Config_Vehicles.hpp#L162)
1to1

**Comment by WolfSkin1337** [comment]
I have resolved the conflicts. It is Updated

**Comment by Jawshy** [comment]
LGTM

**Comment by BoGuu** [comment]
LGTM

**Comment by BoGuu** [comment]
LGTM

**Comment by BoGuu** [comment]
LGTM

**Pull request merged**